src/App.jsx

```diff
@@ -13,34 +16,102 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORTED_BY_ALFABET = 'Sort alphabetically';
+const SORT_FIELD_LENGTH = 'length';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPreparedGoods(goods, { sortField, reverse }) {
```

Comment:
You can move it in a separate file as helper/util func. Then import and use the inside component. ( for future)